scripts/testing/xbox-ssl.conf

```diff
@@ -117,12 +117,12 @@
     # https://xbox.local/dxos/ipfs/webui/* -> http://localhost:5001/webui/*
     RewriteRule ^/dxos/ipfs/webui/(.*) http://localhost:5001/webui/$1 [P,L]
 
-    # /ipfs/ is baked into a lot of IPFS software (and X-Ipfs-Path response) but we move that to /dxos/ipfs/gateway
-    RewriteRule ^/ipfs/?(.*) /dxos/ipfs/gateway/$1 [R]
```

Comment:
Why are we removing this and adding the localhost rule below?

Reply:
Moving from redirecting to proxying the request.  The problem is that some of the IPFS tooling just plain expects /ipfs/ to work.  We were basically still staking a claim to /ipfs/ by having the redirect rule, but since not all tools chase redirects, proxying makes more sense.